nixos/modules/services/hardware/asusctl.nix

```
@@ -0,0 +1,482 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.asusctl;
+
+  localAsusdConfig = {
+    bat_charge_limit = 100;
+    panel_od = false;
+  };
+
+  localAuraConfig = {
+    brightness = "Med";
+    current_mode = "Static";
+    builtins = { };
+    multizone = null;
+    multizone_on = false;
+    enabled = {
+      AuraDev19b6 = [
+        "ShutdownLogo"
+        "BootBar"
+        "AwakeBar"
+        "SleepBar"
+        "AwakeLogo"
+        "ShutdownBar"
+        "SleepLogo"
+        "SleepKeyb"
+        "BootLogo"
+        "ShutdownKeyb"
+        "AwakeKeyb"
+        "BootKeyb"
+      ];
+    };
+  };
+
+  localAnimeConfig = {
+    system = [ ];
+    boot = [
+      {
+        ImageAnimation = {
+          file = "${cfg.package}/usr/share/asusd/anime/custom/sonic-run.gif";
```

```suggestion
          file = "${cfg.package}/share/asusd/anime/custom/sonic-run.gif";
```
Can we drop this from the package?

---

Done I removed the default.